app/models/provider.rb

```diff
@@ -1,5 +1,5 @@
 class Provider < Base
-  has_many :courses, param: :provider_code
+  self.primary_key = :provider_code
```

Comment:
Why are we removing the has_many association?

Reply:
Sorry, it's not relevant to this PR at all, shame on me. I added it in the precursor to this PR but realised that it's not used anywhere.